packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -585,9 +585,15 @@ export class InternalStateProvider implements CSP.IChainStateService {
   }
 
   private extractAddress(address: string): string {
-    const extractedAddress = address
-      .replace(/^(bitcoincash:|bchtest:|bitcoin:)/i, '')
-      .replace(/\?.*/, '');
+    const extractedAddress = address.replace(/^(bitcoincash:|bchtest:|bitcoin:)/i, '').replace(/\?.*/, '');
```

Comment:
can we just do `address.replace(^[a-z]*:/i,'')` [...]   ?

Reply:
We could, I think this was just a whitespace change